ivy/container/losses.py

```diff
@@ -11,8 +11,8 @@ class ContainerWithLosses(ContainerBase):
     def static_cross_entropy(
         true: Union[ivy.Container, ivy.Array, ivy.NativeArray],
         pred: Union[ivy.Container, ivy.Array, ivy.NativeArray],
-        axis: Optional[int] = -1,
-        epsilon: Optional[float] = 1e-7,
+        axis: Optional[Union[int, ivy.Container]] = -1,
```

Comment:
these new changes are great, but again I've noticed as miss-use of the `Optional` type hint, which should only be used for arguments with default value of `None`.